.gitignore

```diff
@@ -42,3 +42,4 @@ celerybeat-schedule.db
 .coverage
 .tox
 .cache
+*.sublime-*
```

Comment:
Many apologies, I was on my phone.  Thank you for being so patient with me!  I'll make a new PR and mention this one for history.